### ISSUE TITLE
Support for Crystal 0.34?

### ISSUE THREAD

**Issue by confact** [arcage/ip2country#2]
Any thought of adding support for 0.34?

Now I get this error:
```
In lib/ip2country/src/ip2country/ip2cc.cr:92:92
@mtime = File.exists?(@cache_file) ? File.info(@cache_file).modification_time : Time.new(2000, 1, 1)
                                                                                       ^--
Error: wrong number of arguments for 'Time.new' (given 3, expected 0..2)
```

**Comment by confact** [comment]
Also, get this error when it tries to cache everything:
```
Unhandled exception: Arithmetic overflow (OverflowError)
  from lib/ip2country/src/ip2country/ipaddr.cr:42:35 in 'octet'
  from lib/ip2country/src/ip2country/ipaddr.cr:46:22 in 'octets'
  from lib/ip2country/src/ip2country/ipaddr.cr:54:11 in 'to_s'
```
This happen when I changed all `Time.new` to `Time.utc`

**Comment by arcage** [comment]
Thank you for reporting.

I will check it at whit weekend.

**Comment by arcage** [comment]
OK, I released v0.1.4 to fix it.
Check it!

**Comment by confact** [comment]
Works good! Thanks!
